sveltekit/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { sveltekit } from '@sveltejs/kit/vite';
+import mkcert from 'vite-plugin-mkcert'
+
+/** @type {import('vite').UserConfig} */
+const config = {
+    plugins: [sveltekit(), mkcert()],
+    server: {
+        https: true
+    },
+    optimizeDeps: {
+        include: ["axios"],
```

Comment:
@roberto-butti can we maybe remove axios here was well since it is not needed as a dependency anymore? :)

Reply:
removing ....

Reply:
done